corgietc/corgietc.py

```diff
@@ -11,6 +11,8 @@
 from EXOSIMS.util._numpy_compat import copy_if_needed
 import astropy.constants as const
 import scipy.interpolate
+from tqdm import tqdm
+from scipy.optimize import minimize, root_scalar
 
 
```

Comment:
Need to add: 

1. Default values in __init__ 
2. Check of RefStar_SpectralType in __init__ before adding value to default_vals_extra2
3. Check for presence of values in mode (and population of defaults) in populate_observingModes_extra
4. Add to Class docstring.